yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -461,3 +461,17 @@ def test_magnetic_field_aliasing():
                   'magnetic_field_divergence']:
         assert ('gas',field) in ds.derived_field_list
         ad[('gas',field)]
+
+output_00080 = "output_00080/info_00080.txt"
+@requires_file(output_00080)
+def test_field_accession():
+    ds = yt.load(output_00080)
+    fields = [
+        ('gas', 'density'),  # basic ones
+        ('gas' ,'pressure'),
+        ('gas', 'pressure_gradient_magnitude'), # requires ghost zones
```

Comment:
Wouldn't it be equivalent and more direct to test for `density_gratient_magnitude` instead ?
My understanding is that you need a gradient field to test the ghost zones' effect, but if pressure is a derived field then you're actually testing two things at once (and maybe that's a good thing tbh, I don't have a strong opinion, just curious)

Reply:
So it probably is slightly better to test `pressure_gradient_magnitude`. The reason for this is that reading "ghost zones" requires reading from file and not from memory, as some cells may be in boundary regions, or refined, etc.
So when testing `pressure_gradient_magnitude`, we're testing the gradient is properly computed _and_ the underlying pressure as well, including in cells that would normally not be read.

Does this make sense?

Reply:
You’re right, it causes less redundancy.